src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -2350,24 +2356,24 @@ private Matches() {}
     });
   }
 
-  public static Match<Unit> unitCanScrambleOnRouteDistance(final Route route) {
+  public static Predicate<Unit> unitCanScrambleOnRouteDistance(final Route route) {
     return Match.of(unit -> UnitAttachment.get(unit.getType()).getMaxScrambleDistance() >= route.getMovementCost(unit));
   }
 
-  static Match<Unit> unitCanIntercept() {
+  static Predicate<Unit> unitCanIntercept() {
     return Match.of(u -> UnitAttachment.get(u.getType()).getCanIntercept());
   }
 
-  static Match<Unit> unitCanEscort() {
+  static Predicate<Unit> unitCanEscort() {
     return Match.of(u -> UnitAttachment.get(u.getType()).getCanEscort());
   }
 
-  static Match<Unit> unitCanAirBattle() {
+  static Predicate<Unit> unitCanAirBattle() {
     return Match.of(u -> UnitAttachment.get(u.getType()).getCanAirBattle());
   }
 
-  static Match<Territory> territoryIsOwnedByPlayerWhosRelationshipTypeCanTakeOverOwnedTerritoryAndPassableAndNotWater(
-      final PlayerID attacker) {
+  static Predicate<Territory>
+  territoryIsOwnedByPlayerWhosRelationshipTypeCanTakeOverOwnedTerritoryAndPassableAndNotWater(final PlayerID attacker) {
```

Comment:
This is a Checkstyle indentation violation.  Try:

```java
  static Predicate<Territory> //
      territoryIsOwnedByPlayerWhosRelationshipTypeCanTakeOverOwnedTerritoryAndPassableAndNotWater(
          final PlayerID attacker) {
```

The trailing comment on the first line will prevent the formatter from re-joining the first two lines.